fuse_loss/fuse_plugins.xml

```diff
@@ -0,0 +1,72 @@
+<library path="lib/libfuse_loss">
+  <class type="fuse_loss::TrivialLoss" base_class_type="fuse_core::Loss">
+    <description>
+    This corresponds to no robustification. It is not normally necessary to use this, as setting not loss function
```

Comment:
I summarized what I believe it's the important information from the ceres documentation, mainly from the source code comments: https://github.com/ceres-solver/ceres-solver/blob/master/include/ceres/loss_function.h

Reply:
:smiley: